lib/git/config.go

```diff
@@ -35,10 +42,12 @@ func CompileAncestorBranches(branchName string) (result []string) {
 // DeleteAllAncestorBranches removes all Git Town ancestor entries
 // for all branches from the configuration.
 func DeleteAllAncestorBranches() {
-	configs := util.GetCommandOutput("git", "config", "--get-regexp", "^git-town-branch\\..*\\.ancestors$")
-	for _, config := range strings.Split(configs, "\n") {
-		splitConfig := strings.Split(config, " ")
-		removeConfigurationValue(splitConfig[0])
+	configRegexp, err := regexp.Compile("^git-town-branch\\..*\\.ancestors$")
+	if err != nil {
+		log.Fatal("Error compiling ancestor branches configuration regular expression: ", err)
+	}
+	for _, key := range getConfigurationKeysMatching(configRegexp) {
+		removeConfigurationValue(key)
```

Comment:
Had to change how this works as `git config --get-regexp "^git-town-branch\\..*\\.ancestors$"` returns an error code if there are it matches no keys. This was surfaced in one of the feature tests for rename-branch.

Same applies to a similar code block below